### PR TITLE
Remove local dev banner when doing e2e tests

### PR DESCRIPTION
**Pull request by zhx828**
This is a part of fixes for https://github.com/cBioPortal/cbioportal/issues/7561

### CODE REVIEW

**Comment by alisman** [comment]
@inodb i think we could write that test to directly check the frontendConfig.frontendUrl variable

**Comment by alisman** [comment]
@zhx828 based on screenshots it doesn't look like this worked.  i think because .local-dev-banner is not actually applied to div.   unfortunately that div is created in load_frontend.js of backend repo.  Annoying.  We might as well just move that to frontend.  I mean the creation of that div.

**Comment by zhx828** [comment]
@alisman I added the class at the backend. We need to merge the backend to master and deploy before merging the frontend repo. https://github.com/cBioPortal/cbioportal/pull/7573

**Comment by inodb** [comment]
@alisman checking frontendConfig.frontendUrl sounds good

File: end-to-end-test/remote/specs/core/home.spec.js [hunk 1]
```diff
@@ -21,11 +21,15 @@ describe('homepage', function() {
...
-            assert(browser.getText('.alert-warning').indexOf('dev mode') > 0);
+        it('window.frontendConfig.frontendUrl should point to localhost 3000 when testing', function() {
+            // We no longer check whether the dev mode banner exits.
+            // The banner is hidden in e2etests.scss
+            assert.equal(
+                browser.execute(function() {
+                    return window.frontendConfig.frontendUrl;
+                }).value,
+                '//localhost:3000/'
+            );
```
Comment (zhx828) [inline end-to-end-test/remote/specs/core/home.spec.js:32]: @inodb @alisman here is the updated test. Let me know whether I'm doing it right. Thank you.

Comment (alisman) [reply end-to-end-test/remote/specs/core/home.spec.js:33]: looks right to me

Comment (inodb) [reply end-to-end-test/remote/specs/core/home.spec.js:33]: looks good to me

**Comment by zhx828** [comment]
Three failing tests are not related to this pr.

**Pull request merged**